autoInstaller.sh

```diff
@@ -467,5 +493,5 @@ fi
 
 # Generate ViViaN Documentation
 if $generateViVDox; then
-    $scriptdir/ViViaN/vivianInstall.sh -i $instance -s $scriptdir
+    $scriptdir/ViViaN/vivianInstall.sh -i $instance -s $scriptdir -y $gtmInst
```

Comment:
i think this could be resolved for looking for the gtm/yottadb environment variables by default and leaving the -y as an optional parameter for those installs.

Reply:
I'm going to remove the gtmInst variable and instead look for the presence of the env file for GT.M/YottaDB as the trigger for GT.M instances

if [ -f /home/$instance/etc/env]; then
  <do gtm things>
else
  <do cache things>